extensions/amp-audio/1.0/test/validator-amp-audio.html

```diff
@@ -0,0 +1,36 @@
+<!--
+  Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS-IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the license.
+-->
+<!DOCTYPE html>
+<html ⚡ lang="en">
+  <!-- prettier-ignore -->
+  <head>
+    <meta charset="utf-8">
+    <title>amp-audio example</title>
+    <link rel="canonical" href="/examples/amp-audio.html">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+      amp-audio {
+        color: red;
+      }
+    </style>
+    <script async custom-element="amp-audio" src="https://cdn.ampproject.org/v0/amp-audio-1.0.js"></script>
```

Comment:
For this to be valid, you will need to modify https://github.com/ampproject/amphtml/blob/main/extensions/amp-audio/validator-amp-audio.protoascii. It looks like you've [done that before](https://github.com/ampproject/amphtml/pull/34828/files#diff-118b3b959e3647e36061adf25c225d50af2b83232bf8e5033872e0af95524d04); I think you can follow the same pattern here?

Reply:
Absolutely!

I will finish validator, [preact test-case](https://github.com/ampproject/amphtml/pull/35342#issuecomment-890119179) and [`onPlaying`](https://github.com/ampproject/amphtml/pull/35342#discussion_r676743263) by 5th August, EoD 📝

Reply:
@twifkak Validator test is failing with following:
```
ValidatorRulesMakeSense : amp-audio has html_format either explicitly or implicitly set for AMP4ADS
but amp-audio version 1.0 is not approved for AMP4ADS. If this version is intended for AMP4ADS
please get approval from @wg-monetization and then update this test. Otherwise remove the
version for AMP4ADS from the tagspec
```

Would you please guide me for the correct action or steps necessary from my end?

Reply:
@twifkak ping

Reply:
Sorry for the delay! It looks like you already made the change per [@caroqliu's answer](https://github.com/ampproject/amphtml/pull/35342#discussion_r685237499) which looks good to me.